integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala

```diff
@@ -48,8 +48,13 @@ object CarbonThriftServer {
       System.exit(0)
     }
 
+    val master = Option(System.getProperty("spark.master"))
```

Comment:
Was this the reason for NoClassDefFoundError?

Reply:
No, it's support CarbonThriftServer run in local/cluster, which convenient for users. Before if user want to run CarbonThriftServer in local, user need add master("local")

Reply:
ok

Reply:
I think better to let user give --master local, user can control this.

Reply:
ok, done